dashboards/kubernetes/k8s-pod-prometheus.json

```diff
@@ -161,242 +190,251 @@
+          "pieChart": {
+            "chartType": "PIE",
             "dataSets": [
               {
-                "tableDisplayOptions": {},
+                "dimensions": [],
+                "measures": [],
                 "timeSeriesQuery": {
-                  "prometheusQuery": "sum by (cluster) (kube_pod_status_scheduled)"
+                  "prometheusQuery": "sum by (cluster) (kube_pod_status_scheduled{${Cluster},${Location},${Namespace}})",
+                  "unitOverride": ""
                 }
               }
             ],
-            "metricVisualization": "NUMBER"
-          },
-          "title": "Scheduled Pods Per Cluster"
-        },
-        "width": 4,
-        "xPos": 0,
-        "yPos": 4
+            "sliceAggregatedThreshold": 0
+          }
+        }
       },
       {
-        "height": 4,
+        "xPos": 32,
+        "yPos": 12,
+        "width": 16,
+        "height": 16,
         "widget": {
-          "timeSeriesTable": {
+          "title": "Failed Pods Per Cluster",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
             "dataSets": [
               {
-                "tableDisplayOptions": {},
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "sum by (cluster) (kube_pod_status_phase{phase=\"Failed\"})"
+                  "prometheusQuery": "sum by (cluster) (kube_pod_status_phase{phase=\"Failed\",${Cluster},${Location},${Namespace}})",
+                  "unitOverride": ""
                 }
               }
             ],
-            "metricVisualization": "NUMBER"
-          },
-          "title": "Failed Pods Per Cluster"
-        },
-        "width": 4,
-        "xPos": 4,
-        "yPos": 4
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
       },
       {
-        "height": 4,
+        "xPos": 16,
+        "yPos": 12,
+        "width": 16,
+        "height": 16,
         "widget": {
-          "timeSeriesTable": {
+          "title": "Pods Per Namespace",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
             "dataSets": [
               {
-                "tableDisplayOptions": {},
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "STACKED_AREA",
+                "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "sum by (exported_namespace) (kube_pod_status_scheduled{condition=\"true\"})"
+                  "prometheusQuery": "sum by (exported_namespace) (kube_pod_status_scheduled{condition=\"true\",${Cluster},${Location},${Namespace}})",
```

Comment:
for some reason in my project, this metric does not have a `exported_namespace` label. It should always have `namespace` though I believe. Does it work to group by `namespace` instead?

Reply:
Asked @huygaa11 about this as he's worked more with kube-state-metrics, it _seems_ we should probably use `namespace`
- found a couple questions online where people point to this being a setting https://github.com/prometheus-community/helm-charts/issues/1368. I don't know how common it is either way, but
- the GKE curated kube-state-metrics package (https://cloud.google.com/kubernetes-engine/docs/how-to/configure-metrics#ksm-package) seems to use `namespace`, and this is likely the most common source of kube-state-metrics going forward.

Reply:
This has been changed to `(namespace)` and should be good to go. Hope you're having a great day!